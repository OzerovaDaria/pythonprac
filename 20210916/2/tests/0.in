[111, 89, 7, 0, 27]
