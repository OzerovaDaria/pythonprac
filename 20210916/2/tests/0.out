[0, 7, 27, 89, 111]
